scripts/spack/packages/umpire/package.py

```diff
@@ -54,6 +58,9 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
     patch('camp_target_umpire_3.0.0.patch', when='@3.0.0')
     patch('cmake_version_check.patch', when='@4.1')
     patch('missing_header_for_numeric_limits.patch', when='@4.1:5.0.1')
+    # BEGIN AXOM EDIT
+    patch('cmake_devicealloc_option_6.0.0.patch', when='@6.0.0axom')
```

Comment:
Did this patch get into Umpire? and/or is it from a later version of Umpire?

Reply:
I see part of it here: https://github.com/LLNL/Umpire/blob/e22a12e571a128ef9bc596583c557a74f2eaa4c8/cmake/SetupUmpireOptions.cmake#L38, so I'm assuming it is the latter.